### PR TITLE
How to Configure MySQL Master-Slave Replication on CentOS 7

### PR DESCRIPTION
**Pull request by 1817666**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: MohammadHossein Fakhraei eng.fakhraei@gmail.com

### CODE REVIEW

**Comment by BackInBash** [comment]
It would be nice to add a consistency check like [pt-table-checksum](https://www.percona.com/doc/percona-toolkit/2.0/pt-table-checksum.html).

**Comment by 1817666** [comment]
@BackInBash I think writing a complete tutorial about MySQL consistency check in the future is an interesting idea.

File: tutorials/configure-mariadb-master-slave-replication-on-centos-7/01.en.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+```
+
+```bash
+MariaDB [(none)]> UNLOCK TABLES;
+MariaDB [(none)]> quit;
+```
+
+Copy the exampledb.sql file to your Slave server.
+
+Here, I am going to copy this file to the **/home/** folder of my slave server. So the command will be:
```
Comment (n-se) [inline tutorials/configure-mariadb-master-slave-replication-on-centos-7/01.en.md]: You copied it to the /root/ folder

Comment (1817666) [reply tutorials/configure-mariadb-master-slave-replication-on-centos-7/01.en.md]: Fixed.

**Comment by 1817666** [comment]
> Is the mysql port now world opened or just to this single slave and/or master server? If the port is public now, maybe secure it with iptables or something similar.

This tutorial is a simple Master/Slave Replication.
It is good idea to explain in a separate article about security in MySQL/MariaDB and MySQL consistency check.

**Comment by n-se** [comment]
@blanky0230 Could you please give your opinion here before we merge this?

**Comment by 1817666** [comment]
> @blanky0230 Could you please give your opinion here before we merge this?

@n-se Please check this tutorial again, if have any problem please let me know to fix it.
Thanks.

**Pull request merged**